### ISSUE TITLE
The slick-greeter package is a missing accessibility text at login in terms of username 

### ISSUE THREAD

**Issue by kimariterikishi** [linuxmint/slick-greeter#179]
```
 * slick-greeter version (slick-greeter --version) | All versions
 * Distribution | Linux Mint 20.x/19.x, LMDE 4/Debian 10, Ubuntu 20.04/18.04 (all Linux)
 * Graphics hardware *and* driver used | Does not matter
 * 32 or 64 bit | 64 bit, 32 bit
 ```

**Issue**
For the graphical settings detailed below, at the time of notification, the Orca screen reader will also read the password request text when requesting a username.
My reason is that the slick-greeter package does not include the username text.
Password text is included **only**. This is about:

```
#: src/greeter-list.vala:302
#, c-format
msgid "Enter password for %s"
msgstr "Enter password for %s"
```

Source:
https://github.com/linuxmint/slick-greeter/blob/master/po/en_GB.po

For the Xfce desktop environment, both settings are included (it' good, but, the package is a lightdm-gtk-greeter).
I show, the good choice for username text (-> msgid "**Enter username for %s**"): 

```
#: ../src/lightdm-gtk-greeter.glade.h:11
msgid "Enter your username"
msgstr "Adja meg felhasználónevét"

#: ../src/lightdm-gtk-greeter.glade.h:12
msgid "Enter your password"
msgstr "Adja meg jelszavát"
```

Source:
https://github.com/Xubuntu/lightdm-gtk-greeter/blob/master/po/hu.po

The shortfall should be made up.
The problem is a disturbing phenomenon for the blind and visually impaired.

**Steps to reproduce**

The problem can be caused in this way 

1) The login screen after installing Linux Mint. 
https://i.postimg.cc/CLQnZq9N/slick-greeter-01.png

2) Basic settings.
https://i.postimg.cc/PJ6pQpJd/slick-greeter-02.png

3) Changed settings. The error comes with these. ("Allow manual login" and "Hide the user list" settings enabled).
https://i.postimg.cc/Hsx7s2vn/slick-greeter-03.png

4) After rebooting, Orca will read "Enter password for% s", but "Enter username for% s" would fit here.
https://i.postimg.cc/c1TnSjmP/slick-greeter-04.png

5) The text "Enter password for% s" will be read, and this is fine. 
https://i.postimg.cc/g09wsPzL/slick-greeter-05.png

6) Installed lightdm packages. (There is also an issue with Debian after installing the slick-greeter package, which means that I think the former package should be fixed.) .
https://i.postimg.cc/y83g0NHz/slick-greeter-06.png

**Other information**

It occurs not only in the MATE desktop environment, but in the Cinnamon desktop environment, for example, as the source of the package is the Linux Mint GitHub site. But other Linux distributions have the same problem as Debian GNU / Linux (I use this with the Cinnamon desktop environment).
